### PR TITLE
Fix setting a default theme when editing interactions

### PR DESCRIPTION
**Pull request by rafenden**
## Description of change

In some cases the `theme` field for interactions is not filled, if this happens we should use the `Other` theme by default.

## Test instructions

1. Edit interaction that doesn't have `theme` field filled.
2. The form should not display the `theme` and `kind` step.

### CODE REVIEW

File: test/sandbox/polyfills.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+if (!Object.assign) {
+  Object.defineProperty(Object, 'assign', {
+    enumerable: false,
+    configurable: true,
+    writable: true,
+    value: _.assign,
+  })
+}
```
Comment (adamledwards) [inline test/sandbox/polyfills.js:8]: 👍

File: docs/Running tests.md [hunk 2]
```diff
@@ -60,9 +60,10 @@ Sandbox is as a light replacement for API backend and it's used only by function
 1. Install sandbox, for more info see [instructions](https://github.com/getsandbox/sandbox)
 
 ```bash
-wget https://s3.amazonaws.com/sandbox-binaries/runtime-binary.tar \
-  && tar -C /usr/local/bin -xzvf runtime-binary.tar \
-  && rm runtime-binary.tar
+wget https://github.com/getsandbox/sandbox/releases/download/1.1.0/sandbox-runtime.tar \
```
Comment (adamledwards) [inline docs/Running tests.md:63]: We should update docker image with the same version at some point

File: test/functional/cypress/specs/interaction/interaction-details-form-spec.js [hunk 3]
```diff
@@ -880,3 +881,19 @@ describe('Filtering services based on theme & kind', () => {
...
+
+describe('Editing an interaction without a theme', () => {
+  it('should set a default theme (Other)', () => {
+    cy.visit(urls.interactions.edit(interactionWithoutTheme.id))
+
+    cy.get('#field-subject').then((element) =>
+      assertFieldInput({
+        element,
+        label: 'Subject',
+        value: interactionWithoutTheme.subject,
```
Comment (adamledwards) [inline test/functional/cypress/specs/interaction/interaction-details-form-spec.js:893]: I think you have a string vs int issue in your functional test.

File: docs/Running tests.md (hunk 2 above)
Comment (rafenden) [reply docs/Running tests.md:63]: Good idea, the Dockerfile was updated in all places.

File: test/functional/cypress/specs/interaction/interaction-details-form-spec.js (hunk 3 above)
Comment (rafenden) [reply test/functional/cypress/specs/interaction/interaction-details-form-spec.js:893]: Can you elaborate on that?

File: docs/Running tests.md (hunk 2 above)
Comment (rafenden) [reply docs/Running tests.md:63]: Thanks, done!

File: test/functional/cypress/specs/interaction/interaction-details-form-spec.js (hunk 3 above)
Comment (adamledwards) [reply test/functional/cypress/specs/interaction/interaction-details-form-spec.js:893]: Sorry the problem is not on your test it was a bug by a previous dev. you fixed the incorrect usage of the  nullish coalescing  here test/functional/cypress/support/assertions.js but I think we will need to add here 
https://github.com/uktrade/data-hub-frontend/blob/0043f11597b759e663cf348ced230ce038ff0540/test/functional/cypress/specs/companies/edit-spec.js#L272-L275

Comment (rafenden) [reply test/functional/cypress/specs/interaction/interaction-details-form-spec.js:893]: This is now fixed, thanks!

**Pull request merged**